rotkehlchen/constants/assets.py.template

```diff
@@ -214,4 +214,5 @@ A_SX = EthereumToken('0x99fE3B1391503A1bC1788051347A1324bff41452')
 A_RSPT = EthereumToken('0x016bf078ABcaCB987f0589a6d3BEAdD4316922B0')
 A_FOX = EthereumToken('0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d')
 A_ENS = EthereumToken('0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72')
-A_PSP = EthereumToken('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5')
\ No newline at end of file
+A_PSP = EthereumToken('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5')
+A_SDL = EthereumToken('0xf1dc500fde233a4055e25e5bbf516372bc4f6871')
```

Comment:
The reason the asset is not properly displayed is this initialization. The address is not checksummed and the asset is not correctly resolved. You need to change it to `0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871` and then recreate the `assets.py` from this template